ios/FluentUI.Demo/FluentUI.Demo/DemoColorThemeWindows.swift

```diff
@@ -6,70 +6,74 @@
 import FluentUI
 import UIKit
 
-class DemoColorThemeDefaultWindow: UIWindow, ColorProviding {
-    func primaryColor(for window: UIWindow) -> UIColor? {
-        return Colors.communicationBlue
-    }
-
-    func primaryTint10Color(for window: UIWindow) -> UIColor? {
-        Colors.Palette.communicationBlueTint10.color
-    }
-
-    func primaryTint20Color(for window: UIWindow) -> UIColor? {
-        Colors.Palette.communicationBlueTint20.color
-    }
-
-    func primaryTint30Color(for window: UIWindow) -> UIColor? {
-        Colors.Palette.communicationBlueTint30.color
-    }
+class DemoColorThemeWindow: UIWindow, ColorProviding {
+    public static var theme: NSString = "Default"
```

Comment:
We shouldn't merge all the themes into one. The fundamental concept of FluentUI is that we have a specific theme per window. The demo app serves as an example to be followed and we don't want client apps creating a theme that changes colors instead of setting one per window.

Reply:
The cause of this bug is because of the different windows/pushing view controllers vs. just updating the table. I tried to just force accessibility to behave properly by posting accessibility notifications in my first commit of #619, but @harrieshin wanted to update the table instead of this hackier route. Harrie, thoughts here?

Reply:
can you change the theme color of the window without pushing a new view controller on the stack?

Reply:
That's what I've done in this change. If we keep the original implementation (two different theme windows), I don't think so because changing the theme color would require an entirely new UIWindow and the view controller would have to be pushed onto it somehow.

Reply:
there isn't a way to update the windows colors by having some sort of property so we can still do just updating the tableview based on "right" setting but still keep the infrastructure to have multiple theme per window?

Reply:
If you look at the implementation of the theme windows, the colors are returned as functions of the theme. I don't think it's possible to keep this implementation and show a different color from the window's theme. Even if it were possible, that would mean showing a green theme on a DemoColorThemeDefaultWindow or a default theme on a DemoColorThemeGreenWindow, which doesn't seem right either.

Reply:
right so why can't theme be not static class variable and per instance of windows? and the colors that returns is still based on theme string(or enum)? which will still allow different windows have different themes in iPad.

Reply:
not sure if I understand--if we have different windows with different themes, doesn't that bring us back to having to change windows when user selects a different theme, which would reset the voiceover focus to the back button (initial bug)?

Reply:
can we not change the windows but just refresh the view to grab the new color from the provider?

Reply:
the color provider _**is**_ the window, so if we don't change the window, we can't change the color.

Reply:
the color provider doesn't have to be the window itself. color provider is just a protocol. any object can a color provider.

Reply:
but if we make the view controller the color provider for example, it wouldn't make sense for a green themed view controller to be presented on a DemoColorTheme**Default**Window, right?

Reply:
I don't think the colordemoviewcontroller should be the color provider. color provider should be its own object and depending on the selection of the segment control, couldn't we just set appropriate provider of the existing window in the Colors.setProvider(provider:window:)? I still don't understand why you would set green theme provider in default case?